xc/hotplug.ml

```diff
@@ -136,18 +136,12 @@ let wait_for_plug (task: Xenops_task.task_handle) ~xs (x: device) =
 let wait_for_unplug (task: Xenops_task.task_handle) ~xs (x: device) =
   debug "Hotplug.wait_for_unplug: %s" (string_of_device x);
   try
-    let path = path_written_by_hotplug_scripts x in
```

Comment:
This would be called by Hotplug.release, but AFAICT all callers of Hotplug.release for Vbd devices have called a hard or clean shutdown first, so modifying Hotplug.wait_for_unplug shouldn't be needed for qdisk.